entreprise7pro-bootstrap-sass.gemspec

```diff
@@ -1,6 +1,6 @@
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'entreprise7pro-bootstrap-sass/version'
+require 'bootstrap-sass/version'
```

Comment:
We do not want to include twitter bootstrap-sass, we must keep entreprise7pro-bootstrap-sass.

Reply:
ok with that said, the rake convert should not need MS-IE